packages/nextjs/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "baseUrl": "./",
```

Comment:
For some reason my IDE was not autocompleting the path without adding this to `tsconfig` : 

Without `baseUrL` : 
![Screenshot 2023-03-17 at 4 32 01 PM](https://user-images.githubusercontent.com/80153681/225895438-daa41640-e6a6-4e73-975d-ca961bfb358d.jpg)

With `baseUrl` : 
![Screenshot 2023-03-17 at 4 32 20 PM](https://user-images.githubusercontent.com/80153681/225895504-3f59538d-d087-4545-af49-aba6d2930d23.jpg)

although removed `baseUrl` at [aa15853](https://github.com/scaffold-eth/se-2/pull/246/commits/aa15853099448acaf89d63ec889e9b13179a6acf) since https://github.com/microsoft/TypeScript/pull/40101 and https://github.com/vercel/next.js/pull/34926 and might be problem with my IDE